### DISCUSSION TITLE
get "AdGuard Home cannot configure it automatically for this network interface. Please look for an instruction on how to do this manually."

### DISCUSSION THREAD

**Discussion by johnaaronrose** [AdguardTeam/AdGuardHome#6484] [Q&A]
On my Raspberry Pi model 4 (with Raspbian Desktop installed) when installing AdGuard I get "AdGuard Home cannot configure it automatically for this network interface. Please look for an instruction on how to do this manually.". 
![AdGuard](https://github.com/AdguardTeam/AdGuardHome/assets/397379/8144203b-2131-4ee3-9c15-f247ecb61728)
My Pi has a Static NAT address of 192.168.2.10 given by my dhcp server on my TP-Link router. I have ports 3000/TCP, 80/TCP & 53/TCP/UDP (as I set DoH on my TP-Link router) open on the Pi with Port Forwarding for them to be passed to the Pi.
Have I got the AdGuard parameters correct on the installation screens or is there another reason why I get the above error message?

**Comment by JoshQuint** [comment]
Having the same issue when installing AdguardHome on my Pi4B (default lite OS) and connecting it to my FritzBox (which uses its internal dhcp server) via ethernet (tried wifi too, same result). Giving the pi a static ip, or not doing so, does not seem to have an effect.

@johnaaronrose  Were you able to resolve this?
